src/events/neo4j.js

```diff
@@ -0,0 +1,265 @@
+// All Session and Transaction properties are private
+/* eslint-disable no-underscore-dangle, prefer-rest-params */
+
+/**
+ * @fileoverview: Handlers for Neo4j driver instrumentation
+ */
+
+const uuid4 = require('uuid4');
+
+const moduleUtils = require('./module_utils.js');
+const eventInterface = require('../event.js');
+const utils = require('../utils.js');
+const serverlessEvent = require('../proto/event_pb.js');
+const tracer = require('../tracer.js');
+const errorCode = require('../proto/error_code_pb.js');
+
+/**
+ * Extracts the relevant arguments from provided arguments
+ * @returns {Object} Object of the extracted arguments
+ */
+function getArgsFromFunction(...args) {
+    return {
+        query: args[0],
+        params: args[1],
+    };
+}
+
+
+/**
+ * Gets session's address info from Transaction/Session instance
+ * @param {neo4j.Session|neo4j.Transaction} session A DB session (could be Session of Transaction)
+ * @returns {Object} json with address info { host, port }
+ */
+function getAddressInfo(session) {
+    let port = '7474';
```

Comment:
change host and port to consts

Reply:
They can not be, they might be changed later this function.
should i change the function design?

Reply:
oh oh, fixed :O